### DISCUSSION TITLE
Pure-Maps on sxmo Pinephone: No navigation

### DISCUSSION THREAD

**Discussion by whoeveritis** [rinigus/pure-maps#552] [General]
I installed PureMaps on a PinePhone in PostmarketOS sxmo, which is a alpine linux derivate. Navigation does not work. On Pinephone in Manjaro KDE Plasma, everything works very well. So, there's possibly something wrong in the Alpine Package?

**Comment by rinigus** [comment]
Not sure what you mean by "navigation does not work". Does it show you your location? If it does, what precision? I would expect that something is wrong with location services.

**Comment by whoeveritis** [comment]
it does not show my location at auto-center button. But it finds my address. Terminal mentions on start of app:
https://wolkenmeer.com/index.php/s/7EQA4d3xe2RX9Hc
Switching the profile to 'mixed' allows me to select other maps than default, but messages: Error occured while loading current map. (much more text here) .. loading style failed: HTTP status code 401.
I'll try to install another map app and see what it does maybe? Openstreetmaps says on location: Unknown error acquiring position. But it routes nicely between two points, in GraphHopper, but not in OSRM.
hope this helps ;) and thank you.

**Reply by rinigus** [reply]
If it cannot find our location (as with autocenter) then GPS is either not ready or there is some issue with hardware and drivers integration. Sounds like it is an issue in the underlying OS, not Pure Maps or other maps apps. You should ask for help from OS developers.

**Comment by whoeveritis** [comment]
thank you - now i know a little more.

**Reply by whoeveritis** [reply]
my little investigation led to following: 
In PostmarketOS sxmo or PostmarketOS Plasma, Puremaps don't work.
In Manjaro Plasma, Puremaps work very nicely. 
(someone with IT knowledge may be able to run it on Postmarket, though)

**Reply by rinigus** [reply]
Thanks for looking into it further. This indicates that something is missing in Pinephone OS config and should be fixed by PostmarketOS or upstream developers

**Comment by whoeveritis** [comment]
yes, actually the PostmarketOS flavour of Pinephone, or maybe the Alpine Linux Distro? ManjaroOS's Plasma Distro seems much better, though I can easily trigger a timeout with routing a 600km bicycle trail (just for fun). However I don't know how to reach those developers you mention.

**Reply by rinigus** [reply]
They have IRC (or Matrix?) channel which you could use to get in touch
